packages/ra-core/src/dataProvider/useGetMatching.ts

```diff
@@ -0,0 +1,113 @@
+import { useSelector } from 'react-redux';
+import { CRUD_GET_MATCHING } from '../actions/dataActions/crudGetMatching';
+import { GET_LIST } from '../dataFetchActions';
+import { Pagination, Sort, ReduxState } from '../types';
+import useQueryWithStore from './useQueryWithStore';
+import {
+    getReferenceResource,
+    getPossibleReferenceValues,
+    getPossibleReferences,
+} from '../reducer';
+
+const referenceSource = (resource, source) => `${resource}@${source}`;
+
+/**
+ * Call the dataProvider with a GET_LIST verb and return the result as well as the loading state.
```

Comment:
The comment is the same as useGetList... You should tweak it so that readers understand the difference

Reply:
It is tweaked in the example. GET_MANY is actually sent as GET_LIST